packages/shared/src/components/cards/v1/AdCard.tsx

```diff
@@ -45,7 +45,7 @@ export const AdCard = forwardRef(function AdCard(
               alt="Ad image"
               src={ad.image}
               className={classNames(
-                'z-1 w-full mobileXL:w-56',
+                'z-1 w-full mobileXL:h-28 mobileXL:w-40 mobileXXL:h-auto mobileXXL:w-56',
```

Comment:
I see this `mobileXL:h-28 mobileXL:w-40 mobileXXL:h-auto mobileXXL:w-56` is now in all CardImage component, correct me if I'm wrong. If yes, can we possibly make the change there to be in just one place?

Reply:
done 🙏